SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -206,6 +210,7 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner-mshv.cfg
 /boot/vmlinuz-%{uname_r}
 /boot/efi/vmlinuz-%{uname_r}
 %config(noreplace) /boot/linux-%{uname_r}.cfg
+%config(noreplace) %{_sysconfdir}/default/grub.d/50_mariner_mshv.cfg
```

Comment:
I can't recall off the top of my head, will this install correctly if the `grub.d/` directory is missing? (ie you haven't installed the grub  package)

Reply:
(If it fails, we might still want that grub-def-config or whatever package with the files of interest)

Reply:
That is why I do `mkdir -p %{buildroot}%{_sysconfdir}/default/grub.d` in the %install step to create grub.d if it does not already exist.

Reply:
The %install phase does not affect the install process of the rpm (... I know right?). My concern is the actual install process onto a live system. The %install section just creates the folder in the build staging directory when we are packaging.

Reply:
right... %install sets up the buildroot with the contents described by %files. So easy to mix them up.

Reply:
I'm going with Daniel's proposal to package the grub.d folder with the rpm macros and have dependent packages rely upon the macros both for the folder and for running mkconfig (if we have switched from the legacy boot path).